permission_handler/android/src/main/java/com/baseflow/permissionhandler/PermissionManager.java

```diff
@@ -382,6 +409,10 @@ private void executeIntent(String action, int requestCode) {
         activity.startActivityForResult(intent, requestCode);
     }
 
+    private void executeSimpleIntent(String action, int requestCode) {
+        activity.startActivityForResult(new Intent(action), requestCode);
+    }
```

Comment:
What would be the reason preferring this method over the existing `executeIntent` method?

Since we are having some conflicts with other plugins using the same `requestCode` I am considering updating the `onActivityResult` callback to also check the contents of the data field.

Reply:
(respoding to above in a sec but this one's quick)
You can't launch the `ACTION_NOTIFICATION_POLICY_ACCESS_SETTINGS` with the data field including 
`dat=package:com.example.permissionhandlerexample` since there is no dedicated page for that permission for each application — rather it only exists for all apps on the device that contain the `android.permission.ACCESS_NOTIFICATION_POLICY` permission in the `AndroidManafest`

Below is the error without `executeSimpleIntent`

![image](https://user-images.githubusercontent.com/7013191/121553404-3c361780-c9df-11eb-8ee2-dd89e965bc71.png)

Reply:
Aaa yes of course, that makes sense. Thank you for clarifying.